### PR TITLE
#162 | Debug Data

### LINKED ISSUE #162: Combat - Remove Ticks Counter

### PR DESCRIPTION
**Pull request by demwilson**
* Debugging data only available when debug mode is TRACE.

Closes #162

### CODE REVIEW

File: combat/Combat.tscn [hunk 1]
```diff
@@ -177,7 +177,22 @@ __meta__ = {
...
-[node name="Ticks" type="Label" parent="CanvasLayer/Background"]
+[node name="AnimatedSprite" type="AnimatedSprite" parent="CanvasLayer/Background"]
+position = Vector2( 320, 91 )
+frames = SubResource( 2 )
+animation = "satelite"
+frame = 88
+playing = true
+
+[node name="DebugContainer" type="Container" parent="CanvasLayer"]
+visible = false
```
Comment (beserkzombie) [inline combat/Combat.tscn:188]: Why is this false?

Comment (demwilson) [reply combat/Combat.tscn:188]: Defaulting the debug container to not be visible. We turn it on if you have debug mode `TRACE`.

Comment (beserkzombie) [reply combat/Combat.tscn:188]: Oh. Miss saw. Thought it was visible false on the animation sprite.

**Review by beserkzombie — APPROVED** [review]
Clean, perfect Wilsonian standards

**Pull request merged**